cms/djangoapps/cms_user_tasks/apps.py

```diff
@@ -0,0 +1,19 @@
+"""
+CMS user tasks application configuration
+Signal handlers are connected here.
+"""
+
+from django.apps import AppConfig
```

Comment:
Nit: we should probably get in the habit of automatically adding the `__future__` imports [recommended by OEP-7](http://open-edx-proposals.readthedocs.io/en/latest/oep-0007.html#future-imports) to new files